app/models/meeting_room.rb

```diff
@@ -3,4 +3,7 @@
 class MeetingRoom < ApplicationRecord
   validates :name, presence: true, uniqueness: true
   validates :capacity, presence: true
+
+  has_many :bookings, dependent: :destroy
```

Comment:
In case a Meeting room is removed - we cannot just get rid of all the bookings - that would be unexpected for scheduled meetings. We should have a process to get the meeting moved to another room or notify that the meeting room is no longer available and let the user accordingly shift the booking room. This may not be required now - but can be added in as a Feature in future.

Reply:
ok